lib/internal/loader/ModuleRequest.js

```diff
@@ -2,6 +2,7 @@
 
 const fs = require('fs');
 const internalCJSModule = require('internal/module');
+const CJSModule = require('module');
```

Comment:
I've also hoisted up this require by fixing the circular reference in module.js.